taxcalc/behavior.py

```diff
@@ -54,9 +54,9 @@ def behavior(calc_x, calc_y, update_income=update_income):
     """
 
     # Calculate marginal tax rates for plan x and plan y.
-    mtrX = calc_x.mtr('e00200')
+    mtrX = calc_x.mtr('e00200')[2]
```

Comment:
this seems a bit confusing. Based on the function signature of `mtr`, I would suggest:

```
fica, iit, combined = calc_x.mtr('e00200')
mtrX = combined
```